### ISSUE TITLE
Any chance this can be adapted for use on macOS?

### ISSUE THREAD

**Issue by StardewSweetheart** [Norbyte/bg3se#240]
I'd give anything for there to be a macOS compatible version of this tool.

**Comment by LeonaRadiantDawn** [comment]
Any update?

**Comment by Varriount** [comment]
As far as I know, while it's *possible* for this to be adapted to MacOS, it would require a developer with a device running MacOS to test on (which Norbyte may not have). 

Aside from adapting the code patching logic itself to actually use MacOS APIs, one would also likely need to supply information on where the functions to patch are located in the game's executable code (since it's likely that they are in different locations compared to the Windows executables).

**Comment by Norbyte** [comment]
tl;dr there are currently no plans for Mac version.
